### PR TITLE
Return non-zero exit code when credentials are not successfully acquired

### LINKED ISSUE #167: Cred provider returns exit code 0 even if no ADAL token was acquired
`
Version:  v0.1.20+a5d37185a1a95acb4f0eb13a774f5c23cab99508
`
Even if ADAL token isn't found using Cache, Integrated Auth or Device Code flow, the tool returns with exit code 0.

### PR DESCRIPTION
**Pull request by nayanshah**
Fixes #167

### CODE REVIEW

**Comment by nayanshah** [comment]
@jmyersmsft @satbai

File: CredentialProvider.Microsoft/Resources.resx [hunk 1]
```diff
@@ -445,4 +445,7 @@ Device Flow Authentication Timeout
   <data name="WhileShuttingDown" xml:space="preserve">
     <value>While the plugin was shutting down:</value>
   </data>
+  <data name="CredentialsNotFound" xml:space="preserve">
+    <value>Credentials not found.</value>
```
Comment (jmyersmsft) [inline CredentialProvider.Microsoft/Resources.resx]:
```suggestion
    <value>Unable to acquire credentials.</value>
```

File: CredentialProvider.Microsoft/Resources.resx [hunk 2]
```diff
@@ -127,13 +127,13 @@
     <value>Failed to acquire session token: {0}</value>
   </data>
   <data name="AdalDeviceFlowMessage" xml:space="preserve">
-    <value>ATTENTION: User interaction required.
```
Comment (jmyersmsft) [inline CredentialProvider.Microsoft/Resources.resx]: I'm going to assume these changes are some sort of line ending normalization or something

Comment (nayanshah) [reply CredentialProvider.Microsoft/Resources.resx]: Yup, that got added after I applied the commit suggestion. Should I revert it?

**Pull request merged**